backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/LinkStage.kt

```diff
@@ -176,7 +179,13 @@ internal class LinkStage(val context: Context) {
                 KonanTarget.MACOS_X64 -> "Versions/A/$dylibName"
                 else -> error(target)
             }
-            frameworkLinkerArgs = listOf("-install_name", "@rpath/${framework.name}/$dylibRelativePath")
+            val args = mutableListOf("-install_name", "@rpath/${framework.name}/$dylibRelativePath")
+            when (context.shouldEmbedBitcode()) {
```

Comment:
Shouldn't the same args be passed to linker when compiling the program instead of framework?